do-not-commit/is-comparison-string.py

```diff
@@ -0,0 +1,29 @@
+x = object()
+
+# ruleid:identical-is-comparison
+if x is x:
```

Comment:
Found identical comparison using is. Ensure this is what you intended. ([.common-mistakes.is-comparison-string.identical-is-comparison](https://semgrep.dev/r/.common-mistakes.is-comparison-string.identical-is-comparison))
#
<div>
<a href="https://semgrep.dev">
<img align="right" src="https://r2c.dev/semgrep_icon_text.svg" height=28px alt="" />
</a>

<sub> [How do I suppress this?](https://semgrep.dev/docs/running-rules/#managing-findings) **·** [Report a bug](https://github.com/returntocorp/semgrep-rules/issues/new?assignees=&labels=&template=rule_bug_report.md&title=.common-mistakes.is-comparison-string.identical-is-comparison) **·** [Manage rule](https://semgrep.dev/manage/projects)</sub>
</div>